Makefile

```diff
@@ -237,6 +237,10 @@ build-server: setup-network-config
 ifdef ANY_SYNC_NETWORK
 	@$(eval TAGS := $(TAGS) envnetworkcustom)
 endif
+
```

Comment:
add a description in readme

Reply:
it's on purpose. we don't want to highlight it because it's not secure